watchdog-config/extra-sql-queries.yml

```
@@ -53,58 +79,373 @@ dwh_users_with_insificient_disk:
     - cht_instance:
         usage: "LABEL"
 
-dwh_form_completions_last_60_days:
+feedback_docs:
   query: |
     SELECT
-      form_name AS form_name,
-      sum(load_count - complete_count) AS incomplete_count,
-      sum(complete_count) AS complete_count,
-      AVG(complete_count/load_count::float) AS complete_percentage,
-      forms.partner_name,
-      replace(monitoring_urls.url, 'https://', '') as cht_instance
-    FROM
-      public.app_monitoring_form_completions as forms
+        feedback.partner_name,
+        reported::varchar(255),
+        source,
+        detail,
+        count(*) AS source_count,
+        replace(monitoring_urls.url, 'https://', '') as cht_instance
+    FROM app_monitoring_feedback_docs AS feedback
     LEFT JOIN
-      monitoring_urls ON forms.partner_name=monitoring_urls.partner_name
-    WHERE
-      load_count>0
+        monitoring_urls ON feedback.partner_name = monitoring_urls.partner_name
+    WHERE monitoring_urls.url IS NOT NULL AND reported >= NOW() - INTERVAL '90 days'
     GROUP BY
-      form_name,
-      forms.partner_name,
-      cht_instance
-    ORDER BY
-      cht_instance,incomplete_count DESC
+        feedback.partner_name,
+        feedback.reported,
+        source,
+        detail,
+        cht_instance
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - reported:
+        usage: "LABEL"
+    - source:
+        usage: "LABEL"
+    - detail:
+        usage: "LABEL"
+    - source_count:
+        usage: "GAUGE"
+    - cht_instance:
+        usage: "LABEL"
+
+replication_by_status:
+  query: |
+    SELECT
+        status.partner_name,
+        period_start::varchar(255),
+        replication_success_count,
+        replication_failure_count,
+        replication_denied_count,
+        replication_error_count,
+        replace(monitoring_urls.url, 'https://', '') as cht_instance
+    FROM app_monitoring_replication_by_status AS status
+    LEFT JOIN
+        monitoring_urls ON status.partner_name = monitoring_urls.partner_name
+    WHERE monitoring_urls.url IS NOT NULL AND period_start >= NOW() - INTERVAL '90 days'
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - period_start:
+        usage: "LABEL"
+    - replication_success_count:
+        usage: "GAUGE"
+    - replication_failure_count:
+        usage: "GAUGE"
+    - replication_denied_count:
+        usage: "GAUGE"
+    - replication_error_count:
+        usage: "GAUGE"
+    - cht_instance:
+        usage: "LABEL"
+
+replication_activity:
+  query: |
+    SELECT
+        sms.partner_name,
+        reported::varchar(255),
+        delta_of_doc AS delta_docs,
+        replace(monitoring_urls.url, 'https://', '') as cht_instance
+    FROM app_monitoring_sms_error_and_users_docs_replication AS sms
+    LEFT JOIN
+        monitoring_urls ON sms.partner_name = monitoring_urls.partner_name
+    WHERE monitoring_urls.url IS NOT NULL AND reported::date >= NOW() - INTERVAL '90 days' AND delta_of_doc > 0
```

Minor point, but this, and all other `WHERE` clauses would be a bit easier to read with line breaks (this line is especially wide):

```suggestion
    WHERE 
        monitoring_urls.url IS NOT NULL AND 
        reported::date >= NOW() - INTERVAL '90 days' AND 
        delta_of_doc > 0
```